helm/chart-operator/templates/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         app: {{ .Values.project.name }}
         giantswarm.io/service-type: "managed"
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision }}
```

Comment:
Well this is **exciting**. Upstream removed `$.Release.Time` and ` $.Release.Revision` but then added revision back in https://github.com/helm/helm/pull/6688.

AIUI we use this to trigger fresh pods. Revision should also work for that but needs to be tested.